src/components/PeopleFilters.tsx

```diff
@@ -1,92 +1,127 @@
-export const PeopleFilters = () => {
+import classnames from 'classnames';
+import { SearchLink } from './SearchLink';
+import { getSearchWith } from '../utils/searchHelper';
+
+type Props = {
+  searchParams: URLSearchParams;
+  setSearchParams: (params: URLSearchParams) => void;
+};
+
+export const PeopleFilters: React.FC<Props> = ({
+  searchParams, setSearchParams,
+}) => {
```

Comment:
why do notjust  use useSearchParams inside this component? It's a little inconveniently to pass props which you can get from the hook